tests/test_entrypoint/database/test_add.py

```diff
@@ -0,0 +1,65 @@
+import pathlib
+
+import pytest
+import sqlalchemy
+
+from bluesearch.entrypoint.database.parent import main
+
+
+@pytest.fixture()
+def engine_sqlite(tmpdir):
+    db_url = pathlib.Path(str(tmpdir)) / "database.db"
+    eng = sqlalchemy.create_engine(f"sqlite:///{db_url}")
+
+    # Schema
+    metadata = sqlalchemy.MetaData()
+    sqlalchemy.Table(
+        "articles",
+        metadata,
+        sqlalchemy.Column(
+            "article_id", sqlalchemy.Integer(), primary_key=True, autoincrement=True
+        ),
+        sqlalchemy.Column("title", sqlalchemy.Text()),
+    )
+
+    # Table
+    with eng.begin() as connection:
+        metadata.create_all(connection)
+
+    return eng
+
+
+def test_mysql_not_implemented():
+    with pytest.raises(NotImplementedError):
+        main(["add", "a", "b", "c", "--db-type=mysql"])
+
+
+def test_unknown_parser():
+    with pytest.raises(ValueError, match="Unsupported parser"):
+        main(["add", "dburl", "WrongParser", "path_to_files"])
+
+
+def test_sqlite_cord19(engine_sqlite, jsons_path):
+    # Create a dummy database
+    path_jsons = pathlib.Path(__file__).parent.parent.parent / "data" / "cord19_v35"
```

Comment:
Isn't this equal to the `jsons_path` fixture in the parameters, that's actually unused at the moment?

Reply:
Great point!
Fixed in f689cd9b828858d28d514412dadd594c196d1f23